test/features/ui_authorization_authenticate.feature

```diff
@@ -0,0 +1,27 @@
+Feature: The API to authenticate after authentication using the OTP auth factor.
+
+  Once the end-user provides the user identifier (UIN/VID) and all the required auth challenges to the UI application,
+  this endpoint will be invoked.
+  Supported auth-challenge depends on the integrated authentication server. 
+
+  Request endpoint: POST /authorization/authenticate
+
+  Scenario: Successfully authenticate after authentication using the OTP auth factor
+    Given The end-user wants to authenticate after authenticating using the OTP auth factor
+    When The end-user tries to trigger an action with the required parameters
+    Then The end-user successfully authenticates after authenticating using the OTP auth factor
+
+  Scenario: The user is not able to authenticate after authentication using the OTP auth factor because an invalid transactionId parameter
```

Comment:
Naming suggestions: 

Maybe instead ```because an invalid ... parameter/provided``` use ```because an invalid ... was specified``` 
in scenario like it is in ```Then```